docs/releases/2.7.3.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Welcome to the new Pootle 2.7.3 final
+=====================================
+
+*Released on 20 November 2015*
+
+Bugfix release for 2.7.2.
+
+
+Changes in Requirements
+=======================
```

Comment:
I think we need a rethink in this section.  How about:

Requirements
=====

- Django >= 1.7.10, < 1.8 **(updated)**
- Translate Toolkit >= 1.13.0 **(updated)**
- Python >= 2.7, < 3.0
- Redis >= 2.8.4 **(updated)**
- Django transaction hooks **(new)**
- Unix-based operating system.